contracts/src/rollup/ValidatorWallet.sol

```diff
@@ -6,24 +6,93 @@ pragma solidity ^0.8.0;
 
 import "../challenge/IChallengeManager.sol";
 import "../libraries/DelegateCallAware.sol";
+import "../libraries/IGasRefunder.sol";
+import {IRollupUserAbs} from "./IRollupLogic.sol";
 import "@openzeppelin/contracts/utils/Address.sol";
 import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
 
-contract ValidatorWallet is OwnableUpgradeable, DelegateCallAware {
+error BadArrayLength(uint256 expected, uint256 actual);
+error NotCallerOrOwner(address actual);
```

Comment:
I believe this should be renamed to `NotExecutorOrOwner` to match the naming of the field